README.md

```diff
@@ -39,8 +51,14 @@ make
 Finally, build and run the example app:
 
 ```
-cd examples/hello
-pipenv install
-spin build
+cd examples/hello_world
+../../target/release/spin-python app -o app.wasm
 spin up
 ```
+
+**Note:* In the samples provided in the `examples` directory, the `spin build` command depends on the plugin `py2wasm` being installed. Therefore, to test the locally built `spin-python` binary, either replace the build command in the `spin.toml` to invoke it using spin build.
+
+```
+[component.build]
+command = "../../target/release/spin-python py2wasm app - o app.wasm"
```

Comment:
It might be good to clarify the relationship between `spin-python` and `py2wasm` - I thought they were different names for the same thing but looking at this it appears not?

(Or... looking at the diff on line 55... maybe they are the same and this line is not quite right?)

Reply:
That was a typo on my part. I will fix it in the next commit to this PR